scripts/json_generator.py

```diff
@@ -515,23 +515,10 @@ def beginFile(self, genOpts):
 
         self.isCTS = genOpts.isCTS
 
-        self.baseTypeListMap  = {
-                                  "int32_t"   : "deInt32" if self.isCTS else "int32_t",
-                                  "uint32_t"  : "deUint32" if self.isCTS else "uint32_t",
-                                  "uint8_t"   : "deUint8" if self.isCTS else "uint8_t",
-                                  "uint64_t"  : "deUint64" if self.isCTS else "uint64_t",
-                                  "float"     : "float",
-                                  "int"       : "int",
-                                  "double"    : "double",
-                                  "int64_t"   : "deInt64" if self.isCTS else "int64_t",
-                                  "uint16_t"  : "deUint16" if self.isCTS else "uint16_t",
-                                  "char"      : "char"
-                                }
-
         write(headerGuardTop, file=self.outFile, end='')
         write(copyright, file=self.outFile)
         if self.isCTS:
-            write(predefinedCode % ("deUint32", "deUint32"), file=self.outFile)
+            write(predefinedCode % ("uint32_t", "uint32_t"), file=self.outFile)
         else:
             write(predefinedCode % ("uint32_t", "uint32_t"), file=self.outFile)
```

Comment:
These two branches can just be collapsed now.

Reply:
Thanks for fixing it, I added a small update to bake these in `predefinedCode`